common/e2e.go

```diff
@@ -24,7 +24,8 @@ import (
 )
 
 const (
-	customMetadataFile string = "custom-prow-metadata.json"
+	// CustomMetadataFile is the name of the custom metadata file generated for spyglass visualization.
+	CustomMetadataFile string = "custom-prow-metadata.json"
```

Comment:
I know this was already a constant but I wonder if this should be an entry in the config struct....

Reply:
This one is hardcoded in ci-operator -- if a user overrode it it wouldn't end up showing up in Spyglass.